### ISSUE TITLE
add pref setting to "always allow" full screen view

### ISSUE THREAD

**Issue by bradleyrichter** [brave/browser-laptop#5979]
## Test Plan:
1. go to about:preferences#security
2. find new fullscreen select setting, hit "always allow"
3. go to https://vimeo.com, open fullscreen
4. fullscreen should be open without notification prompt 
5. Go back to fullscreen setting
6. Disable with "always ask"
7. Go to https://vimeo.com or any other video website
8. Make video fullscreen, notification should open

**QA note:**

1. When notification for fullscreen is set to "deny", even without "remember this decision" checkbox marked, option is stored and you can't undo. Bug wasn't introduced by this PR and was filled under #7010 with proper STR.

## original issue text 
Adding this option will allow a user to override the default for full-screen requests.

The default will still remain as "Always ask".

![image](https://cloud.githubusercontent.com/assets/13509546/20819432/17409534-b7e9-11e6-906d-8f8c7fa95435.png)

Menu choices:

"Always ask"
"Always allow"

**Comment by bradleyrichter** [comment]
I'll make a separate issue for changes to the alert bar which are waiting on this new setting.

![image](https://cloud.githubusercontent.com/assets/13509546/20819557/eb475a48-b7e9-11e6-9ae7-dd599325720c.png)

**Comment by bsclifton** [comment]
@cezaraugusto would you be up for grabbing this one?

**Comment by abhibeckert** [comment]
Related, the "Example.com entered full screen mode. Press ESC to exit" message is annoying:

 * I know I'm in full screen mode – I just clicked an "enter full screen" button.
 * I know how to exit full screen – I've been doing it for years
 * The message is large and covers part of the page content and cannot be hidden, all you can do is wait for it to go away

I think the message should be removed completely. What reason is there to keep it?
